theme.json

```diff
@@ -179,7 +179,13 @@
             }
           ]
         }
-      }
+      },
+      "core/image": {
+				"lightbox": {
+          "enabled": false,
+					"allowEditing": false
+				}
+			}
```

Comment:
Looks like you are adding tabs here for indendation 😁 Check out [editorconfig](https://greenpeace-gpi.slack.com/archives/C0160MX64AG/p1729754027541789)

Reply:
Fixed!